motoman_hc20_support/package.xml

```diff
@@ -17,11 +17,17 @@
       <li>HC20 - Default</li>
       <li>HC20DT - Default</li>
       <li>HC20XP - Default</li>
+      <li>HC20DTP</li>
+      <li>HC30PL</li>
```

Comment:
Seeing as we noticed there are actual variants of variants in other recently merged support packages: is there nothing we can state here as to which specific specification were used?

Reply:
For those robots, there are only one version specified in the referenced manuals for each model.